cEpiabm/src/population_factory.cpp

```diff
@@ -36,7 +36,7 @@ namespace epiabm
 
     void PopulationFactory::addCell(PopulationPtr population)
     {
-        population->cells().push_back(Cell());
+        population->cells().emplace_back(population->cells().size());
```

Comment:
Not really a comment but more a learning thing for me - why is this better than `push_back()`?

Reply:
push_back would construct the Cell (as an rvalue), then copy it into the vector.
emplace_back constructs the Cell in place.